README.md

```diff
@@ -238,28 +259,31 @@ describe(‘MyComponent’, function() {
 
 ### Performance Considerations
 
- * Connections in separate modules, load what you need.
- * Abortable requests.
- * Optionally cold connections.
+ * Connections in separate modules, load what you need
+ * Abortable requests
+ * Optionally cold connections
 
 ### Accessibility Considerations
 
-### User Experience Considerations
+### End User Experience Considerations
+
+ * More insight into connection state to provide realtime status to users
+ * Better support of streaming data
+ * Better resilience in network flakes via `retry()`
 
-More insight into connection state to provide realtime status to users.
 
 ### Mobile Considerations
 
 ### Caching
 
-The cache requirements are different for HTTP in Angular 2 vs $http in Angular 1, since there is a
+The cache requirements are different for Http in Angular 2 vs $http in Angular 1, since there is a
 separate fetching mechanism for template loading. In Angular 1, every request for a component’s
 template would make an http request that would try to grab the template from the $templateCache
 service before making a network request. And even though Angular 2 supports fetching of modules,
-which AngularJS 1 did not, these fetches will rely on ES6 System.import (or traceur equivalent), not
-the HTTP library.
+which AngularJS 1 did not, these fetches will rely on ES6 `System.import` (or traceur equivalent),
```

Comment:
The System object was removed from the draft a while ago. It (mostly) seems to live in Reflect.Loader now, which was moved into a WHATWG doc. This doesn't matter a whole lot here, but might be good to write "es6 module loading" or something instead